src/shared/components/case-editor/case-edit-confirm/case-edit-confirm.component.ts

```diff
@@ -34,4 +41,17 @@ export class CaseEditConfirmComponent {
   getCaseId(): String {
```

Comment:
should this not be a `string` not `String`

Reply:
updated to string